services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -433,15 +439,15 @@ private void doDeleteFlow(final CommandMessage message, String replyToTopic, Des
      * @param message NetworkCommandData
      */
     private void doNetworkDump(final CommandMessage message) throws SwitchOperationException {
-        logger.debug("Processing dump switches request");
+        logger.info("Processing request from WFM to dump switches. {}", message.getCorrelationId());
```

Comment:
correlationId is logger via MDC. Here we must log switch_id.

Reply:
here we dump **all** switches. So we do not have switchId.

Reply:
Sorry my fail.